background_scripts/main.coffee

```diff
@@ -130,7 +130,7 @@ TabOperations =
     tabConfig.openerTabId = request.tab.id if canUseOpenerTabId
 
     chrome.tabs.create tabConfig, (tab) ->
-      callback extend request, {tab, tabId: tab.id}
+      callback extend request, {tab, tabId: tab.id, position: "", active: false}
```

Comment:
Why are these extra arguments needed? For Chrome they must be no-ops, assuming Chrome's current behavior isn't changed. So if these are important for Firefox, document why.

Reply:
OKay, I've added two comments in code.